cassandra/rab.py

```diff
@@ -29,8 +29,8 @@ class RAB(object):
     def __init__(self, cap_tbl, comm=MPI.COMM_WORLD):
         self.cap_tbl = cap_tbl
         self.comm = comm
-        self.rank = comm.Get_rank()
-        self.status = 0         # status indicator follows ComponentBase convention
+        self.rank = comm.Get_rank() 
+        self.status = 1         # Always show RAB status as successful
```

Comment:
Continuing with the comment above, I don't love this.  What if the RAB fails in some way.  We don't have any programmed failures, but there is always the possibility of a built-in exception being raised.  One possibility is that because the RAB only runs in MP mode (unlike the other components), we can trap exceptions internally and call `MPI_Abort` if we get one.